### ISSUE TITLE
Error deploying widgets from Landsat8 Explorer app

### ISSUE THREAD

**Issue by kv5784** [Esri/Imagery-Apps#4]
I tried to deploy widgets from Landsat8 Explorer app on WAB and the error appeared. This occurs with some of the widgets i.e. Mask, ISCompare and ISChangDetection.
When I tried adding ISChangeDetection, this error appeared:
![ISChangeDetection_issue](https://user-images.githubusercontent.com/74392730/99205341-dc7de200-27ea-11eb-8d3e-a0283e9d1d57.jpg)
From my understanding, it can't find .png image in the following page right?
Please help correct me if I'm wrong.

When I tried adding Mask and  ISCompare widget, another error appeared:
_Error: scriptError: ./resourceLoad.js?wab_dv=2.16_
From what I understand, there's something wrong with the script.
![ISCompare_issue](https://user-images.githubusercontent.com/74392730/99205414-0fc07100-27eb-11eb-807c-24a5b661200a.jpg)

The steps of adding widget is as followed: https://enterprise.arcgis.com/en/portal/latest/use/add-custom-widgets.htm
Please be notified that I am no programmer.
Does additional script have to be written in order for this to work?
Or is it possible to deploy just widgets, but not the app itself (Landsat8 Explorer)?
Please help advise on this.

**Comment by priya4991** [comment]
@kv5784 it is better if you use the widgets from https://github.com/Esri/WAB-Image-Services-Widgets instead of taking widgets from an existing app, because the widgets may be customized a bit for the corresponding apps.

**Comment by kv5784** [comment]
@priya4991 Are there ways to import then to use in WAB/WAB Developer. Or the only way to do this is to alter the code in order to use them on WAB/WAB developer? 
I believe that the widgets implemented in this app is similar to those from https://github.com/Esri/WAB-Image-Services-Widgets but not quite the same. Some of the widgets i.e. Mask, Time Selector in these two apps are different.  They are customized to use particularly with Landsat8 data. Am I understanding this right?

**Comment by priya4991** [comment]
@kv5784 yes some of the widgets may have been customized to work better with the app in question. Hence it is better to use the widgets in the WAB repository for a more generalized version.
